lib/player.rb

```diff
@@ -0,0 +1,60 @@
+require 'awesome_print'
+require_relative '../lib/scoring.rb'
+require 'pry'
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :total
```

Comment:
If you don't want the user accessing it, **don't** make `total` an `attr_reader`  You have a `total_score` method, use that instead.